test/e2e/traces_test.go

```diff
@@ -0,0 +1,245 @@
+//go:build experimentalintegration
+// +build experimentalintegration
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/efficientgo/e2e"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+)
+
+// OTel trace collector that receives in HTTP w/o security, but exports in gRPC with security.
+const (
+	otelForwardingConfig = `
+receivers:
+    otlp:
+      protocols:
+        http:
+            endpoint: 0.0.0.0:4318
+        grpc:
+            endpoint: 0.0.0.0:4317
+
+# extensions:
```

Comment:
will the parts that are commented out be used at some point? if not, I think we can remove them?